tasks/deploy/instances/steth.ts

```diff
@@ -0,0 +1,117 @@
+import { task, types } from "hardhat/config";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { z } from "zod";
+import {
+  DeployInstanceParams,
+  PoolConfig,
+  PoolDeployConfig,
+  zInstanceDeployConfig,
+} from "./schema";
+
+dayjs.extend(duration);
+
+// Set the base token to always be the ETH letant.
```

Comment:
What does `letant` mean?

Reply:
I think a letant might be someone who gets overzealous with find-replace 🤪